examples/deployContractUDC/main.go

```diff
@@ -0,0 +1,172 @@
+// https://sepolia.voyager.online/tx/0x9bc6f6352663aafd71a9ebe1bde9c042590d8f3c8c265e5826274708cf0133
+
+func main() {
+	fmt.Println("Starting deployContractUDC example")
+
+	// Load variables from '.env' file
+	rpcProviderUrl := setup.GetRpcProviderUrl()
+	accountAddress := setup.GetAccountAddress()
+	accountCairoVersion := setup.GetAccountCairoVersion()
+	privateKey := setup.GetPrivateKey()
+	publicKey := setup.GetPublicKey()
+
+	// Initialize connection to RPC provider
+	client, err := rpc.NewProvider(rpcProviderUrl)
+	if err != nil {
+		panic(fmt.Sprintf("Error dialing the RPC provider: %s", err))
+	}
+
+	// Here we are converting the account address to felt
+	accountAddressInFelt, err := utils.HexToFelt(accountAddress)
+	if err != nil {
+		panic(err)
+	}
+
+	// Initialize the account memkeyStore (set public and private keys)
+	ks := account.NewMemKeystore()
+	privKeyBI, ok := new(big.Int).SetString(privateKey, 0)
+	if !ok {
+		panic("Fail to convert privKey to bitInt")
+	}
+	ks.Put(publicKey, privKeyBI)
+
+	fmt.Println("Established connection with the client")
+
+	// Set maxFee
+	maxfee, err := utils.HexToFelt("0x9184e72a000")
+	if err != nil {
+		panic(err)
+	}
+
+	// Initialize the account
+	accnt, err := account.NewAccount(client, accountAddressInFelt, publicKey, ks, accountCairoVersion)
+	if err != nil {
+		panic(err)
+	}
+
+	// Get the accounts nonce
+	nonce, err := accnt.Nonce(context.Background(), rpc.BlockID{Tag: "latest"}, accnt.AccountAddress)
+	if err != nil {
+		setup.PanicRPC(err)
+	}
+
+	// Build the InvokeTx struct
+	InvokeTx := rpc.InvokeTxnV1{
+		MaxFee:        maxfee,
+		Version:       rpc.TransactionV1,
+		Nonce:         nonce,
+		Type:          rpc.TransactionType_Invoke,
+		SenderAddress: accnt.AccountAddress,
+	}
+
+	// Convert the contractAddress from hex to felt
+	contractAddress, err := utils.HexToFelt(UDCAddress)
+	if err != nil {
+		panic(err)
+	}
+
+	// Build the functionCall struct, where :
+	FnCall := rpc.FunctionCall{
+		ContractAddress:    contractAddress,                               //contractAddress is the contract that we want to call
+		EntryPointSelector: utils.GetSelectorFromNameFelt(contractMethod), //this is the function that we want to call
+		Calldata:           getUDCCalldata(accountAddress),                //change this function content to your use case
+	}
+
+	// Building the Calldata with the help of FmtCalldata where we pass in the FnCall struct along with the Cairo version
+	InvokeTx.Calldata, err = accnt.FmtCalldata([]rpc.FunctionCall{FnCall})
+	if err != nil {
+		panic(err)
+	}
+
+	// Sign the transaction
+	err = accnt.SignInvokeTransaction(context.Background(), &InvokeTx)
+	if err != nil {
+		panic(err)
+	}
+
+	// After the signing we finally call the AddInvokeTransaction in order to invoke the contract function
+	resp, err := accnt.AddInvokeTransaction(context.Background(), InvokeTx)
+	if err != nil {
+		setup.PanicRPC(err)
+	}
+
+	time.Sleep(time.Second * 3) // Waiting 3 seconds
```

Comment:
Nit: lets print something like "waiting 3 seconds before querying the transaction status"